java/delete-features-feature-service/src/main/java/com/esri/arcgisruntime/samples/deletefeaturesfeatureservice/MainActivity.java

```diff
@@ -0,0 +1,238 @@
+import java.util.concurrent.ExecutionException;
+
+import android.annotation.SuppressLint;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.MotionEvent;
+import android.view.View;
+import android.widget.Toast;
+
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.data.Feature;
+import com.esri.arcgisruntime.data.FeatureEditResult;
+import com.esri.arcgisruntime.data.FeatureQueryResult;
+import com.esri.arcgisruntime.data.QueryParameters;
+import com.esri.arcgisruntime.data.ServiceFeatureTable;
+import com.esri.arcgisruntime.geometry.GeometryEngine;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.GeoElement;
+import com.esri.arcgisruntime.mapping.view.Callout;
+import com.esri.arcgisruntime.mapping.view.DefaultMapViewOnTouchListener;
+import com.esri.arcgisruntime.mapping.view.IdentifyLayerResult;
+import com.esri.arcgisruntime.mapping.view.MapView;
+
+public class MainActivity extends AppCompatActivity implements ConfirmDeleteFeatureDialog.OnButtonClickedListener {
+
+  private static final String TAG = MainActivity.class.getSimpleName();
+
+  private MapView mMapView;
+
+  private ServiceFeatureTable mFeatureTable;
+
+  private FeatureLayer mFeatureLayer;
+
+  @SuppressLint("ClickableViewAccessibility")
+  @Override protected void onCreate(@Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    mMapView = findViewById(R.id.mapView);
+
+    // create a map with streets basemap
+    ArcGISMap map = new ArcGISMap(Basemap.Type.STREETS, 40, -95, 4);
+
+    // create service feature table from URL
+    mFeatureTable = new ServiceFeatureTable(getString(R.string.feature_layer_url));
+
+    // create a feature layer from table
+    mFeatureLayer = new FeatureLayer(mFeatureTable);
+
+    // add the layer to the ArcGISMap
+    map.getOperationalLayers().add(mFeatureLayer);
+
+    mMapView.setOnTouchListener(new DefaultMapViewOnTouchListener(this, mMapView) {
+      @Override public boolean onSingleTapConfirmed(MotionEvent event) {
+        // create a point from where the user clicked
+        android.graphics.Point point = new android.graphics.Point((int) event.getX(), (int) event.getY());
+
+        // create a map point from a point
+        Point mapPoint = mMapView.screenToLocation(point);
+
+        // for a wrapped around map, the point coordinates include the wrapped around value
+        // for a service in projected coordinate system, this wrapped around value has to be normalized
+        Point normalizedMapPoint = (Point) GeometryEngine.normalizeCentralMeridian(mapPoint);
```

Comment:
do we have to use a wrapped around service for this sample?

Reply:
I guess not. I'd seen it in the Java sample so assumed it was needed.